### PR TITLE
refactor(@vtmn/css-rating): improve accessibility with keyboard + screen reader

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Description

- The previous version of the rating componet wasn't intuitive/accessible with keyboard + screen reader.
- The component has been reworked to be accessible

## Does this introduce a breaking change?

- No (non class name changes)
- Some aria props should be removed
- The class `vtmn-rating--disabled` can be removed and replaced by `aria-disabled="true"`

❤️ Thank you!

### CODE REVIEW

File: packages/showcases/css/stories/components/rating/examples/interactive.html [hunk 1]
```diff
@@ -5,65 +5,166 @@
...
+      <input
+        type="radio"
+        name="rating2"
+        value="5"
+        id="rating2-5"
+        aria-label="5 stars out of 5"
+      />
+      <label for="rating2-5"></label>
     </div>
-    <span class="vtmn-rating_comment--primary" aria-label="Article rating"
```
Comment (lauthieb) [inline packages/showcases/css/stories/components/rating/examples/interactive.html:62]: Why did you delete aria labels for comments?

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks Gaspard!
+ you need to don't allow focus when disabled

**Comment by lauthieb** [comment]
@GaspardMathon We also have a problem with `:hover` state:
Jan-10-2022 17-09-40
It works with values upper than the current value but not with before

File: packages/showcases/css/stories/components/rating/examples/interactive.html (hunk 1 above)
Comment (GaspardMathon) [reply packages/showcases/css/stories/components/rating/examples/interactive.html:62]: I remobed this aria-label because according MDN : https://developer.mozilla.org/fr/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Aria-label should be set up when the text is not visible. Here the text is visible so we shouldn't use it.

**Comment by lauthieb** [comment]
as we still have issues in this PR (discussed together), I convert again to draft.

**Comment by GaspardMathon** [comment]
> @GaspardMathon We also have a problem with `:hover` state: Jan-10-2022 17-09-40 It works with values upper than the current value but not with before.

Resolved in last commit

**Review by lauthieb — APPROVED** [review]
